tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDK.java

```diff
@@ -770,64 +776,80 @@ private Properties getProperties( File file )
 
     public Map<String,Object> getBuildProperties() throws CoreException
     {
+        return getBuildProperties(false,true);
+    }
+
+    public Map<String,Object> getBuildProperties(final boolean reload, final boolean verified) throws CoreException
```

Comment:
verified means validate the items about server setting in build properties. If these values won't correct then we won't load them to memory. Reload means we will load the changed items after we validated the new server setting content and made sure they were correct. What's meaning of avoided?